db/migrate/20230709163809_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :author, null: false, foreign_key: { to_table: 'users' }
+      t.references :post, null: false, foreign_key: true
+
```

Comment:
- [ ] In accordance to the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) given, the like table is directly associated with the user table. Kindly correct this. ℹ️ 

```suggestion
      t.references ::user, null: false, foreign_key: true
      t.references :post, null: false, foreign_key: true
```